stubs/swiftclient/client.pyi

```diff
@@ -55,7 +56,7 @@ class Connection(object):
         self,
         container: str,
         obj: str,
-        contents: Optional[AnyStr],
+        contents: Optional[BinaryIO],
```

Comment:
Why did this change? Was it incorrect previously? As we discussed in person, previously the type checker wasn't actually checking the calls to this function, but now it is.

Reply:
@spiralman It wasn't not correct. `put_object` accepts a string, a file-like object or an iterable to read object data from. The way that `load_from_file` was written, it opened the file and sent the file contents (BinaryIO) and `object.put_object` to `retry.attempt`, and in doing so, the file contents were being converted to a string. When we removed the `retry.attempt` wrapper from `load_from_file`, mypy blew up, stating that `BinaryIO` was not `AnyStr`. 

I will update the stub file to be `contents: Optional[Union[AnyStr , BinaryIO, Iterable]],` so that it more closely resembles what `put_object` actually accepts